<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>tools | Payam Emami</title> <meta name="author" content="Payam Emami"> <meta name="description" content="Some of the public tools i have developed"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?e0c9ab38b75a69b9dc06cc23c13da8dd"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://payamemami.github.io/tools/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Payam </span>Emami</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item active"> <a class="nav-link" href="/tools/">tools<span class="sr-only">(current)</span></a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">tools</h1> <p class="post-description">Some of the public tools i have developed</p> </header> <article> <h1 id="metaboigniter">MetaboIGNITER</h1> <p><a href="https://nf-co.re/metaboigniter" rel="external nofollow noopener" target="_blank">Metaboigniter</a> is bioinformatics pipeline for pre-processing of mass spectrometry-based metabolomics data. The workflow performs MS1 based quantification and MS2 based identification using combination of different modules.</p> <p>This pipeline ingests raw mass spectrometry data in mzML format, typically in the form of peak lists and MS2 spectral data, for comprehensive metabolomics analysis. The key stages involve centroiding, feature detection, adduct detection, alignment, and linking, which progressively refine and align the data. The pipeline can also perform requantification to compensate for missing values and leverages MS2Query for compound identification based on MS2 data, outputting a comprehensive list of detected and potentially identified metabolites.</p> <h1 id="metabolinden">Metabolinden</h1> <p><a href="https://github.com/PayamEmami/nf-core-metabolinden" rel="external nofollow noopener" target="_blank">Metabolinden</a> is bioinformatics pipeline for pre-processing of mass spectrometry-based metabolomics data. It differs from MetaboIGNITER in a way that it allows users to iteratively run a set of parameters to select the most stable one based on internal standards and repeated injection of the same sample.</p> <h1 id="metaboraid">MetaboRAID</h1> <p><a href="https://github.com/metaboraid/metaboraid" rel="external nofollow noopener" target="_blank">MetaboRAID</a> is an R package enabling rapid metabolite identification using mass spectrometry MS1 and MS2 data. MetaboRAID includes functions for estimating adducts, neutral masses, and merging of spectra.</p> <h1 id="mirnatargetdetector">mirnatargetdetector</h1> <p><a href="https://github.com/PayamEmami/mirnatargetdetector" rel="external nofollow noopener" target="_blank">mirnatargetdetector</a> is Nextflow-based pipeline that uses Miranda and RNAHybrid to detect miRNA targets.</p> <h1 id="mixedcirc">Mixedcirc</h1> <p><a href="https://github.com/PayamEmami/mixedcirc" rel="external nofollow noopener" target="_blank">Mixedcirc</a> uses mixed models to perform differential circadian rhythm analysis. It includes options to perform circadian rhythm on various types of data including RNA-seq, RRBS methylation, metabolomics, proteomics etc. Various signal processing and statistical analysis of circadian rhythm detection are supported out of the box.</p> <h1 id="proteomics-rke">proteomics-rke</h1> <p><a href="https://github.com/PayamEmami/proteomics-rke" rel="external nofollow noopener" target="_blank">proteomics-rke</a> is a cloud-based infrastructure that provides a simple way of deploying a k8s cluster to run Nextflow-based workflows.</p> <h1 id="r-kubernetes">r-kubernetes</h1> <p><a href="https://github.com/PayamEmami/r-kubernetes" rel="external nofollow noopener" target="_blank">r-kubernetes</a> is a simple wrapper that can be used to offload R sessions to Kubernetes.</p> <h1 id="integrative_overrepresentation">integrative_overrepresentation</h1> <p><a href="https://github.com/PayamEmami/integrative_overrepresentation" rel="external nofollow noopener" target="_blank">integrative_overrepresentation</a> used p-value fusion to perform pathway-level data integration for a number of modalities where KEGG pathway exists.</p> <h1 id="distance_matrix">distance_matrix</h1> <p><a href="https://github.com/PayamEmami/distance_matrix" rel="external nofollow noopener" target="_blank">distance_matrix</a> is a collection of methods to calculate dissimilarity matrix on a large dataset. It can run on multiple cores and has managame memory footprint.</p> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Payam Emami. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>